### ISSUE TITLE
Add possibility to enter specific compiler options in casual-make-files

### ISSUE THREAD

**Issue by lazan** [casualcore/casual-make#2]
The reason for this enhancement is to be able to have different compiler option in different makefiles i.e. perhaps disable warnings/errors for dependencies to thirdparty include/libs.

**Comment by ahhud** [comment]
@lazan You can already today set an optional `directive` parameter (list) to Compile function. Isn't that sufficient on some level anyway?

Ex
```
make.Compile( 'source/manager/transform.cpp', directive = ['-Wno-dangling-reference'])
```

**Comment by lazan** [comment]
Yeah... If we need it for a whole cmk file, we probably want the directive in general anyway.
